### PR TITLE
Add missing word "requiring"

### PR DESCRIPTION
**Pull request by ryecroft**
Or insisting/mandating/needing

### CODE REVIEW

File: src/introduction/what-is-jj-and-why-should-i-care.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ I know that sounds like a huge claim, but I believe that the rest of this
 tutorial will show you why.
 
 There's one other reason you should be interested in giving `jj` a try: it has
-a `git` compatible backend, and so you can use `jj` on your own, without anyone
-else you're working with to convert too. This means that there's no real
+a `git` compatible backend, and so you can use `jj` on your own, without requiring anyone
+else you're working with convert too. This means that there's no real
```
Comment (steveklabnik) [inline src/introduction/what-is-jj-and-why-should-i-care.md:26]: adding requiring is good, but we also need to not remove this too

**Review by steveklabnik — CHANGES_REQUESTED** [review]

**Review by ryecroft — COMMENTED** [review]
Sounds good 👍

File: src/introduction/what-is-jj-and-why-should-i-care.md (hunk 1 above)
Comment (steveklabnik) [reply src/introduction/what-is-jj-and-why-should-i-care.md:26]:
```suggestion
else you're working with convert to. This means that there's no real
```

**Review by steveklabnik — APPROVED** [review]
Thanks!

**Pull request merged**